packages/hooks-react/src/useProtectedMedia.ts

```diff
@@ -7,5 +8,20 @@ import useContentProtection from './useContentProtection';
 export default function useProtectedMedia(item: PlaylistItem) {
   const apiService = getModule(ApiService);
 
-  return useContentProtection('media', item.mediaid, (token, drmPolicyId) => apiService.getMediaById(item.mediaid, token, drmPolicyId));
+  const [isGeoBlocked, setIsGeoBlocked] = useState(false);
+  const queryResult = useContentProtection('media', item.mediaid, (token, drmPolicyId) => apiService.getMediaById(item.mediaid, token, drmPolicyId));
```

Comment:
Minor suggestion (`queryResult` is a bit generic):
```suggestion
  const contentProtectionQuery = useContentProtection('media', item.mediaid, (token, drmPolicyId) => apiService.getMediaById(item.mediaid, token, drmPolicyId));
```